### DISCUSSION TITLE
Unknown message (ID 52) | HMS-2000-4T + openDTU Fusion

### DISCUSSION THREAD

**Discussion by jgmrs** [tbnobody/OpenDTU#2439] [Q&A] [answered]
Hi there,

I have a PV system based on a Hoymiles HMS-2000-4T micro-inverter, which is having production issues. In order to troubleshoot it I bought an openDTU Fusion board (v2.3) and flashed it with the latest openDTU release (v24.11.7). I'm able to communicate properly with the micro-inverter, however, after issuing a reboot command I'm always getting an unknown message (ID 52), which doesn't even appear in the manual. Has anyone ever faced this issue?

I leave a few screenshots below.

Many thanks

![Screenshot from 2024-11-30 11-32-24](https://github.com/user-attachments/assets/a5e6e534-65d3-4ec1-866e-55404c8f2582)
![Screenshot from 2024-11-30 11-17-06](https://github.com/user-attachments/assets/1199e03c-6b84-45aa-b331-3e9ddd7ca2bd)

**Comment by stefan123t** [comment, marked as answer]
Event 52 is Unknown, even in the S-Miles Installer App this Event code is not documented.
Please refer to our list of known Events from the S-Miles Installer App warn_code.json in https://github.com/tbnobody/OpenDTU/issues/121#issuecomment-1752124208

**Reply by jgmrs** [reply]
Many thanks for your reply. In the meantime I've triggered the warranty procedure :disappointed:

**Reply by stefan123t** [reply]
I am not sure if there is a guarantee by the manufacturer Hoymiles that you can use OpenDTU to manage its devices. You may only qualify for warranty if your inverter can not be restarted by an original Hoymiles DTU Pro/Lite S though.

I was just answering based on our knowledge and the state of our OpenSource Research. There is no guarantee that we can give, that all the features of the latest firmware are known / implemented.
